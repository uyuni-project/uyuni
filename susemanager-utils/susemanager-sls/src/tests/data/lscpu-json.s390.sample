Architecture:                    s390x
CPU op-mode(s):                  32-bit, 64-bit
Byte Order:                      Big Endian
CPU(s):                          2
On-line CPU(s) list:             0,1
Thread(s) per core:              1
Core(s) per socket:              1
Socket(s) per book:              1
Book(s) per drawer:              1
Drawer(s):                       2
NUMA node(s):                    1
Vendor ID:                       IBM/S390
Machine type:                    2964
CPU dynamic MHz:                 5000
CPU static MHz:                  5000
BogoMIPS:                        3033.00
Hypervisor:                      z/VM 6.4.0
Hypervisor vendor:               IBM
Virtualization type:             full
Dispatching mode:                horizontal
L1d cache:                       256 KiB
L1i cache:                       192 KiB
L2d cache:                       4 MiB
L2i cache:                       4 MiB
L3 cache:                        64 MiB
L4 cache:                        480 MiB
NUMA node0 CPU(s):               0,1
Vulnerability Itlb multihit:     Not affected
Vulnerability L1tf:              Not affected
Vulnerability Mds:               Not affected
Vulnerability Meltdown:          Not affected
Vulnerability Spec store bypass: Not affected
Vulnerability Spectre v1:        Mitigation; __user pointer sanitization
Vulnerability Spectre v2:        Mitigation; execute trampolines
Vulnerability Srbds:             Not affected
Vulnerability Tsx async abort:   Not affected
Flags:                           esan3 zarch stfle msa ldisp eimm dfp edat etf3eh highgprs te vx sie
