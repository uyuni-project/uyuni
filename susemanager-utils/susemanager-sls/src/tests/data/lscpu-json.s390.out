{
    "cpu_vendor": "IBM/S390",
    "cpu_numanodes": "1",
    "cpu_cores": "1",
    "cpu_sum": "2",
    "cpu_threads": "1"
}
